test/server/router.spec.js

```diff
@@ -23,27 +23,27 @@ describe('/api', () => {
     const res = await chai.request(server.listen()).get('/api/')
     expect(res.status).toEqual(404)
     expect(res.type).toEqual('application/json')
-    expect(res.body.code).toEqual(errors.NOT_FOUND_ROUTE)
+    expect(res.body.code).toEqual(errors.NOT_FOUND)
     done()
   })
   test('GET /invalid', async done => {
     const res = await chai.request(server.listen()).get('/api/invalid')
     expect(res.status).toEqual(404)
     expect(res.type).toEqual('application/json')
-    expect(res.body.code).toEqual(errors.NOT_FOUND_ROUTE)
+    expect(res.body.code).toEqual(errors.NOT_FOUND)
     done()
   })
   test('GET /[students|users]', async done => {
     const resStudents = await chai.request(server.listen()).get('/api/students')
     expect(resStudents.status).toEqual(200)
     expect(resStudents.type).toEqual('application/json')
     expect(resStudents.body).toBeDefined()
-    expect(resStudents.body !== { code: errors.NOT_FOUND_ROUTE }).toBeTruthy()
+    expect(resStudents.body !== { code: errors.NOT_FOUND }).toBeTruthy()
```

Comment:
Comparar dois objetos nunca é bom.

`expect(rest.student.body.code).not.toBe(errors.NOT_FOUND)`

https://jestjs.io/docs/en/expect.html#toequalvalue

Reply:
TOPER

Reply:
tá aí